monitor/src/main/webapp/WEB-INF/tiles/skeleton.jsp

```diff
@@ -73,7 +73,7 @@
          
         </nav>
         
-        <h1><a class="appName" href="<c:url value="/" />">Monitor</a> <span style="margin-left: 15px;" class="small">How are your sites are doing?</span></h1>
+        <h1><a class="appName" href="<c:url value="/" />">Monitor</a> <span style="margin-left: 15px;" class="small">How are your sites doing?</span></h1>
```

Comment:
Alright, I'll test it with that just so we can get this done with, but PLEASE from now on keep your changes contained to their own branches!

Reply:
I'm sorry about this. I did a git pull incorrectly and it merged mon3 with this branch.